app/controllers/organisation_session_controller.rb

```diff
@@ -0,0 +1,15 @@
+class OrganisationSessionController < ApplicationController
```

Comment:
Do you think this should live inside the `Users` namespace along with `SessionsController`?